CMakeLists.txt

```diff
@@ -12,60 +12,98 @@
 
 # Set minimum CMake version
 cmake_minimum_required(VERSION 2.8.4)
-
-# Set efm32-base project location
-set(BASE_LOCATION .)
```

Comment:
the reason for `BASE_LOCATION` instead off `CMAKE_CURRENT_LIST_DIR` here is that you might include `efm32-base` as a submodule (or in a subfolder) so there needs to be a method for resolving the components in whatever directory they exist.

Reply:
:+1: